src/System.Net.HttpListener/src/System/Net/CookieExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+
+namespace System.Net
+{
+    internal static class CookieExtensions
+    {
+        private static Func<Cookie, string> s_toServerStringFunc;
+
+        public static string ToServerString(this Cookie cookie)
+        {
+            if (s_toServerStringFunc == null)
+            {
+                s_toServerStringFunc = (Func<Cookie, string>)typeof(Cookie).GetMethod("ToServerString").CreateDelegate(typeof(Func<Cookie, string>));
+            }
+            return s_toServerStringFunc(cookie);
+        }
```

Comment:
@weshaggard, FYI on another reflection case to access internals in another assembly.